wasm/webapi/esm-integration/resources/worker.js

```diff
@@ -1 +1,5 @@
-import * as mod from "./worker.wasm"
+import * as mod from "./worker.wasm";
+import source modSource from "./worker.wasm";
+assert_true(modSource instanceof WebAssembly.Module);
+assert_true(await import("./worker.wasm") === mod);
+assert_true(await import.source("./worker.wasm") === modSource);
```

Comment:
Lets split this in two files/tests, for implementations that support only one and not the other.